packages/conf/flags.go

```diff
@@ -126,6 +126,9 @@ var (
 
 	// UpdateInterval is interval in seconds for checking updates
 	UpdateInterval = flag.Int64("updateInterval", defaultUpdateInterval, "Interval in seconds for checking updates, default 3600 seconds (1 hour)")
+
+	// NoCheckReadAccess disables the access check for reading, is used only for VDE
+	NoCheckReadAccess = flag.Bool("noCheckReadAccess", false, "Do not check access for reading, only used for VDE")
```

Comment:
use of inverse logic blows my mind. I think it would be better to make flag like CheckReadAccess with true by default.